inginious/frontend/parsable_text.py

```diff
@@ -216,7 +218,21 @@ def html(cls, string, show_everything=False, translation=gettext.NullTranslation
         out, _ = tidylib.tidy_fragment(string)
         return out
     
-    
+    def from_dict(self, dictionary, show_everything=False, translation=gettext.NullTranslations()):
+        """Parses DICT"""
+        # Load object
+
+        grader_results = dictionary["grader_results"]
```

Comment:
Just in case could be:
```suggestion
        grader_results = dictionary.get("grader_results",{})
```